### ISSUE TITLE
use my own .ckpt

### ISSUE THREAD

**Issue by zhaobingbingbing** [hiveml/tensorflow-grad-cam#5]
I hope to use this work to Visualize my own classification model.At first,I use pretrained model resnetv2_152 l to finetune my dataset , and I got my own model.ckpt,I use this model.ckpt replaced ‘flags.DEFINE_string("checkpoint_path"./imagenet/resnetv2_152’, and I replaced  label.txt to my own label.txt. But there are some proplems,
**"InvalidArgumentError (see above for traceback): Assign requires shapes of both tensors to match. lhs shape= [13] rhs shape= [12]
	 [[Node: save/Assign_810 = Assign[T=DT_FLOAT, _class=["loc:@resnet_v2_152/logits/biases"], use_locking=true, validate_shape=true, _device="/job:localhost/replica:0/task:0/gpu:0"](resnet_v2_152/logits/biases, save/RestoreV2_810/_1)]]
"** 
Do you where am I wrong?

**Comment by ryanjay0** [comment]
Try setting "label_offset" to 0.  Imagenet on ResNet has a background class and requires the 1.
